collector/src/lib.rs

```diff
@@ -698,10 +707,16 @@ impl<V: vdaf::Collector> Collector<V> {
                 None => None,
             };
 
-            let backoff_duration = if let Some(duration) = backoff.next_backoff() {
+            let backoff_duration = if let Some(duration) = backoff.next() {
                 duration
             } else {
-                // The maximum elapsed time has expired, so return a timeout error.
+                // Either the maximum retries or the maximum delay has been reached
+                // so return a timeout.
+                warn!(
```

Comment:
I'm fine taking these `warn!`s back out, but they were super valuable for debugging timeouts here.

Reply:
If they're useful for debugging, let's move them to `debug!`.

Reply:
Fixed in 9ecf65b784efbded5487e8d19ee56345c7b5e2ef